### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by soreau**

### CODE REVIEW

File: wayfire.desktop.in [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Version=0.4
+Name=Wayfire Session
+Comment=Use this session to run Wayfire as your desktop environment
+Exec=$PREFIX/bin/startwayfire
+Icon=$PREFIX/share/wayfire/icons/wayfire.png
```
Comment (soreau) [inline wayfire.desktop.in:6]: The icon here is installed by wf-shell but maybe wayfire should install it instead?

Comment (ammen99) [reply wayfire.desktop.in:6]: Yes, that would make sense. I didn't realize this inconsistency.

**Review by ammen99 — CHANGES_REQUESTED** [review]
The idea was to generate an uninstall script which is generated when installing. This way, we can be sure that the prefix is the same in both install and uninstall.

Also, this would make sure our script doesn't get very complicated. I would like to keep everything simple, if possible.

**Comment by soreau** [comment]
> The idea was to generate an uninstall script which is generated when installing. This way, we can be sure that the prefix is the same in both install and uninstall.
> 
> Also, this would make sure our script doesn't get very complicated. I would like to keep everything simple, if possible.

I simply removed the commit which adds uninstall because I think it belongs in a separate PR. So this is ready to merge from my end, it doesn't depend on moving wayfire.png from wf-shell to wayfire.

File: install.sh [hunk 2]
```diff
@@ -171,3 +170,14 @@ if [ $yn = Y ]; then
...
     $SUDO ninja -C build install
 fi
+
+SESSIONS_DIR=/usr/share/wayland-sessions
+ask_confirmation "Do you want to install wayfire.desktop to $SESSIONS_DIR/ [y/n]?"
+if [ $yn = Y ]; then
+    cp $BUILDROOT/wayfire.desktop.in $BUILDROOT/wayfire.desktop
+    sed -i "s@^Exec.*@Exec=$PREFIX/bin/startwayfire@g" $BUILDROOT/wayfire.desktop
+    sed -i "s@^Icon.*@Icon=$PREFIX/share/wayfire/icons/wayfire.png@g" $BUILDROOT/wayfire.desktop
+    $SUDO cp $BUILDROOT/wayfire.desktop $SESSIONS_DIR
```
Comment (ammen99) [inline install.sh]: Use `install -m 644` instead of just `cp`.

**Review by ammen99 — COMMENTED** [review]
LGTM aside from the comment.

**Review by ammen99 — APPROVED** [review]
Thanks, and sorry for the delay.

**Pull request merged**